modules/sildisco/tests/AddIdpTest.php

```diff
@@ -105,23 +109,24 @@ public function testAddIdp2NameId_GoodString()
      */
     public function testAddIdp2NameId_GoodArray()
     {
+        $nameID = new NameID();
+        $nameID->setValue('Tester1_SmithA');
+        $nameID->setFormat('urn:oasis:names:tc:SAML:1.1:nameid-format:transient');
```

Comment:
In other places, you change the format from SAML 1.1 transient to SAML 2.0 persistent. Do you want/need to do that here as well?

Reply:
I will change it for consistency. Somewhere in the upgrade process, a change was made that seems to cause the format to be forced to 2.0 persistent. So the value set at the beginning of this test seems to be ignored.